grakn-engine/src/main/java/ai/grakn/engine/SystemKeyspaceImpl.java

```diff
@@ -150,12 +110,18 @@ public boolean deleteKeyspace(Keyspace keyspace){
         return true;
     }
 
-    /**
-     * Load the system schema into a newly created system keyspace. Because the schema
-     * only consists of types, the inserts are idempotent and it is safe to load it
-     * multiple times.
-     */
-    public void loadSystemSchema() {
+    @Override public Stream<Keyspace> keyspaces() {
```

Comment:
Does this actually work? We returning a stream which depends on the transaction being open and if the stream is executing lazily it might do so with a closed transaction?

Reply:
Good point! I think I always immediately collect the result. I'll return a `Set` instead.